.github/workflows/ci.yml

```diff
@@ -37,10 +37,12 @@ jobs:
             toxenv: py38
           - python: pypy3
             toxenv: pypy3
+          #- python: 3.x
+          #  toxenv: py3-master
+          #- python: 3.x
+          #  toxenv: py3-devel
```

Comment:
I guess you can remove the lines. I'll add them again when working on libyang 2 support.